pyperformance/_benchmark.py

```diff
@@ -164,6 +164,10 @@ def runscript(self):
     def extra_opts(self):
         return self._get_metadata_value('extra_opts', ())
 
+    @property
+    def install_setup(self):
+        return self._get_metadata_value('install_setup', False)
```

Comment:
Looking back on [my previous review](https://github.com/python/pyperformance/pull/197#pullrequestreview-985170156), I still think it would be better to provide the path to the setup.py file, if there is one, or `None` otherwise.

```suggestion
    @property
    def setup_py(self):
        if not self._get_metadata_value('install_setup', False):
            return None
        filename = os.path.join(os.path.dirname(self.metafile), 'setup.py'))
        return filename if os.path.exists(filename) else None
```

The fact that pip only supports passing the directory isn't something we should design around here.  We can handle that quirk in the code where we interact with pip.

Reply:
You're suggesting we just pass `dirname(filename)` to pip?  That would work.